core/src/main/java/org/apache/gravitino/tag/TagManager.java

```diff
@@ -431,27 +424,7 @@ private TagEntity updateTagEntity(TagEntity tagEntity, TagChange... changes) {
   // for this entity, with this uid tags can be associated with this entity.
   // This method should be called out of the tree lock, otherwise it will cause deadlock.
   @VisibleForTesting
-  boolean checkAndImportEntity(String metalake, MetadataObject metadataObject, GravitinoEnv env) {
-    NameIdentifier entityIdent = MetadataObjectUtil.toEntityIdent(metalake, metadataObject);
-    Entity.EntityType entityType = MetadataObjectUtil.toEntityType(metadataObject);
-
-    switch (entityType) {
-      case METALAKE:
-        return env.metalakeDispatcher().metalakeExists(entityIdent);
-      case CATALOG:
-        return env.catalogDispatcher().catalogExists(entityIdent);
-      case SCHEMA:
-        return env.schemaDispatcher().schemaExists(entityIdent);
-      case TABLE:
-        return env.tableDispatcher().tableExists(entityIdent);
-      case TOPIC:
-        return env.topicDispatcher().topicExists(entityIdent);
-      case FILESET:
-        return env.filesetDispatcher().filesetExists(entityIdent);
-      case COLUMN:
-      default:
-        throw new IllegalArgumentException(
-            "Unsupported metadata object type: " + metadataObject.type());
-    }
+  void checkAndImportEntity(String metalake, MetadataObject metadataObject, GravitinoEnv env) {
+    MetadataObjectUtil.checkMetadataObject(metalake, metadataObject, env);
   }
```

Comment:
The `checkAndImportEntity(...)` function only have one line code.
We can directly call `MetadataObjectUtil.checkMetadataObject(metalake, metadataObject, env)`.

Reply:
This is convenient to test. Because we can spy this method easily.

Reply:
Fixed.